tests/ecmp/test_fgnhg.py

```diff
@@ -267,6 +267,33 @@ def validate_packet_flow_without_neighbor_resolution(ptfhost, duthost, ip_to_por
     assert neigh_resolved
 
 
+def setup_static_neighbor_entry(duthost, ip, mac, prefix_list):
+    """
+    Performs addition of static entries of ipv4 and v6 neighbors in DUT
+    """
+    if isinstance(ipaddress.ip_network(prefix_list[0]), ipaddress.IPv4Network):
+        logger.info("adding ipv4 static arp entry for ip %s on DUT" % (ip))
+        duthost.shell("sudo arp -s {0} {1}".format(ip, mac))
+    else:
+        logger.info("adding ipv6 static arp entry for ip %s on DUT" % (ip))
+        duthost.shell("sudo ip -6 neigh replace {0} lladdr {1} dev Vlan{2}".format(ip, mac, DEFAULT_VLAN_ID))
+
+
+def link_startup(duthost, ip_to_port, prefix_list, shutdown_link):
+    """
```

Comment:
Can you help me understand why this `link_startup` is needed?

Reply:
We found from 202205 and newer version. When testing shut down an interface, the neighbor/Arp tables will lose Permanent. And we need to add back the neighbor entry when starting up that interface.